go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/autoscaling v1.26.0
 	github.com/aws/aws-sdk-go-v2/service/cloudformation v1.25.0
 	github.com/aws/aws-sdk-go-v2/service/cloudtrail v1.22.0
-	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.18.0
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.19.0
```

Comment:
is it expect to have `cloudwatchlogs` changes here?

Reply:
This is not directly related to the changes for this PR, but the build process also updates SDK dependencies.